### PR TITLE
Ticket Number: Blah blah blah some title about the issue/improvement

### PR DESCRIPTION
**Pull request by SaadZellag**
This is a very good description that shows what is a PR

### CODE REVIEW

File: Website game 1/Assets/ConnectToServer.cs [hunk 1]
```diff
@@ -3,11 +3,21 @@
 using UnityEngine;
 using Photon.Pun;
 using UnityEngine.SceneManagement;
+using Photon.Realtime;
 
 public class ConnectToServer : MonoBehaviourPunCallbacks
 {
+    public static string userId = null;
```
Comment (Zonum9) [inline Website game 1/Assets/ConnectToServer.cs:10]: me no like

File: Website game 1/Assets/ConnectToServer.cs [hunk 2]
```diff
@@ -3,11 +3,21 @@
...
+
```
Comment (Zonum9) [inline Website game 1/Assets/ConnectToServer.cs:11]: `public static string userId = none;`

File: Website game 1/Assets/ConnectToServer.cs [hunk 3]
```diff
@@ -3,11 +3,21 @@
 using UnityEngine;
 using Photon.Pun;
 using UnityEngine.SceneManagement;
+using Photon.Realtime;
 
 public class ConnectToServer : MonoBehaviourPunCallbacks
 {
```
Comment (Zonum9) [inline Website game 1/Assets/ConnectToServer.cs:9]:
```suggestion
{//add this comment
```

**Review by Zonum9 — CHANGES_REQUESTED** [review]
i requested changes

File: Website game 1/Assets/ConnectToServer.cs (hunk 3 above)
Comment (SaadZellag) [reply Website game 1/Assets/ConnectToServer.cs:9]: no

**Review by Zonum9 — APPROVED** [review]
lgtm Dis gud🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

**Pull request merged**